projects/ngx-skltn/ng-package.json

```diff
@@ -1,7 +1,10 @@
 {
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/ngx-skltn",
+  "whitelistedNonPeerDependencies": [
+    "angular-resize-event"
```

Comment:
@VictorJuliani My package size 2.5kb - [https://bundlephobia.com/result?p=ngx-skltn@2.0.0](https://bundlephobia.com/result?p=ngx-skltn@2.0.0)
And angular-resize-event package 4kb - [https://bundlephobia.com/result?p=angular-resize-event@1.1.1](https://bundlephobia.com/result?p=angular-resize-event@1.1.1)

So you think it's good idea to increase package in 160% ? 😞 

Actually i don't have ability to add vendor deps. Cause we use this package in large app in production.

I think it'll be fine solution to add size check in polling, where we already checking window.href  🤔